frontend/src/components/AnnotationView/AllianceGenomeCard.vue

```diff
@@ -0,0 +1,273 @@
+      // This escape character is very necessary, but eslint doesn't think so
+
+      const regex = new RegExp(`\\[background:\]?(.*)`, 'g'); // eslint-disable-line
+
+      const matchResult = this.model.name.match(regex);
+
+      if (matchResult) {
+        return matchResult[0];
+      }
+
+      return '';
+    },
+    associatedHumanDiseasesStyle() {
+      if (Object.keys(this.associatedHumanDiseasesData).length === 0) {
+        return {color: `var(--rosalution-grey-300)`};
+      }
+
+      return 'color: black';
+    },
+    associatedPhenotypesStyle() {
+      if (Object.keys(this.associatedPhenotypesData).length === 0) {
+        return {color: `var(--rosalution-grey-300)`};
+      }
+
+      return 'color: black';
+    },
+    experimentalConditionStyle() {
+      if (Object.keys(this.experimentalConditions).length === 0) {
+        return {color: `var(--rosalution-grey-300)`};
+      }
+
+      return 'color: black';
+    },
+  },
+  methods: {
+    calculateAssociatedHumanDiseases() {
+      this.associatedHumanDiseasesData = this.model.diseaseModels;
+    },
+    calculateAssociatedPhenotypes() {
+      const phenotypesDict = {'':
+                {
+                  phenotypes: [],
+                  style: '',
+                  icon: '',
+                  iconStyle: {},
+                },
+      };
+
+      this.model.phenotypes.forEach((phenotype) => {
+        phenotypesDict[''].phenotypes.push(phenotype);
+      });
+
+      this.frequentTermsObject.forEach((term) => {
+        phenotypesDict[term.term] = {
+          phenotypes: [],
+          style: term.style,
+          icon: term.icon,
+          iconStyle: term.iconStyle,
+        };
+      });
+
+      this.model.phenotypes.forEach((phenotype) => {
+        this.frequentTermsObject.forEach((term) => {
+          const regex = new RegExp('\\b' + term.term + '\\b', 'i');
+
+          if (phenotype.match(regex)) {
+            phenotypesDict[term.term].phenotypes.push(phenotype);
+            phenotypesDict[''].phenotypes.splice(phenotypesDict[''].phenotypes.indexOf(phenotype), 1);
+          }
+        });
+      });
+
+      Object.keys(phenotypesDict).forEach((key) => {
+        if (!phenotypesDict[key].phenotypes.length) {
+          delete phenotypesDict[key];
+        }
+      });
+
+      this.associatedPhenotypesData = phenotypesDict;
+    },
+    calculateExperimentalCondition() {
+      if (Object.keys(this.model.conditions).length == 0) {
+        return;
+      }
+
+      this.experimentalConditions = this.model.conditions.has_condition;
+    },
+  },
+});
+
+</script>
+
+<style scoped>
+
+div {
+  font-family: "Proxima Nova", sans-serif;
+}
+
+ul {
+  list-style-type: disc;
+  padding-left: 7%;
+}
+
+li {
+  margin-left: 10px;
+  padding-top: 1%;
+  font-size: 15px;
+}
+
+.card-header {
+    font-size: 22px;
+    font-weight: bold;
+}
+
+.card-sub-header {
+    font-size: 16px;
+    padding-top: 2%;
+}
+
+.card-base {
+  position: relative;
```

Comment:
`position: relative` is in here and in line https://github.com/uab-cgds-worthey/rosalution/pull/86/files#diff-4147c803031de44ccebf4f782e3286f623ca5337b299556eebecb515f27bf122R244 244. I disabled them both and it didn't seem to cause any change? So they can both probably be removed? I might be wrong, since I don't have access to my larger monitor now to review to see if it is affected there.

Reply:
`position: relative` has been removed from `.card-base`